amplify/backend/function/addVerifiedMembers/src/graphql.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+  createVerifiedMember: /* GraphQL */ `
+    mutation CreateVerifiedMember($input: CreateVerifiedMemberInput!) {
+      createVerifiedMember(input: $input) {
+        colonyAddress
+        userAddress
```

Comment:
In the schema, the return type for this function is `[VerifiedMember]`, but in reality it's whatever subset of fields you fetch in this mutation. And because one of the fields of `VerifiedMember` is of `User` type, you would also need to query all the fields of the `User` model, and keep updating them whenever they change - a bit of a maintenance hell.

This would fail:
```gql
mutation AddMember($input: AddVerifiedMembersInput!) {
  addVerifiedMembers(input: $input) {
    user {
      address
    }
  }
}
```

Try to avoid making lambdas dependent on GraphQL types as they can unpredictably change, maybe returning a boolean like the other mutation would be a better option?